### PR TITLE
Ability to enable a Twitch Stream embed on the site

### PR DESCRIPTION
**Pull request by hhff**
### Description

Demo below:
https://www.loom.com/share/df0046aa44844658a6e3f475550a0af3

### Type(s) of changes

- [ ] Bug fix
- [x] New feature
- [ ] Update to an existing feature

### CODE REVIEW

**Comment by hhff** [comment]
Aaaand here's how it works for Elie:
https://www.loom.com/share/52b8f242bd77438eaf88c802e7c19c7f

**Review by chrismekim — APPROVED** [review]
Makes the site feel more alive! 🥰

**Comment by chrismekim** [comment]
> Aaaand here's how it works for Elie:
> https://www.loom.com/share/52b8f242bd77438eaf88c802e7c19c7f

I'll add this to the Operator Manual too!

File: theme/layout/theme.liquid [hunk 1]
```diff
@@ -23,12 +23,29 @@
   {% render 'nav' %}
 
   <main role="main">
+    {% if shop.metafields.accentuate.show_twitch_stream %}
+      <div
```
Comment (sepowitz) [inline theme/layout/theme.liquid:27]: I'm cool with this, but would prefer we throw it in a snippet to keep this file small.

**Review by sepowitz — APPROVED** [review]

File: theme/layout/theme.liquid (hunk 1 above)
Comment (hhff) [reply theme/layout/theme.liquid]: Fixed, thank you!

**Comment by hhff** [comment]
> I'll add this to the Operator Manual too!

TY @chrismekim !!

**Pull request merged**